python/dgl/data/chem/tox21.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
+
+from .csv_dataset import CSVDataset
+from .utils import smile2graph
+from ..utils import get_download_dir, download, _get_dgl_url, Subset
+
+try:
+    import pandas as pd
+except ImportError:
+    raise ImportError("Please install pandas")
+
+
+class Tox21(CSVDataset):
+    _urls = 'dataset/tox21.csv.gz'
+
+    def __init__(self, smile2graph=smile2graph):
+        data_path = get_download_dir() + '/tox21.csv.gz'
+        download(_get_dgl_url(self._urls), path=data_path)
+        df = pd.read_csv(data_path)
```

Comment:
Add an option for shuffling the dataset.

Reply:
I prefer user using `split_dataset` function in the main file.